### ISSUE TITLE
Settings editor should have Accessibility TOC entry

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#217810]
Testing #217636

* open settings editor
* make sure TOC is visible
* it has no accessibility entry which IMO it should have

<img width="928" alt="Screenshot 2024-06-25 at 09 56 01" src="https://github.com/user-attachments/assets/61b18a20-7b58-4a3b-ab79-e886b850300f">

**Comment by rzhao271** [comment]
The command `workbench.action.openAccessibilitySettings` is currently the best approach to showing accessibility settings.
The settings currently span multiple categories under the Settings editor and are grouped by a tag, though I do notice an accessibility sub-category.

![accessibility tag in the Settings editor showing accessibility](https://github.com/user-attachments/assets/79006c23-c594-486c-b615-06468df3eb9a)

**Comment by daviddossett** [comment]
I was also surprised to find it under `Features`. 👍 for promoting this to its own top-level entry.

**Comment by meganrogge** [comment]
I think this should be top level also
